res/database/Default/user/therob.xml

```diff
@@ -1997,7 +1997,7 @@
 					</title>
 					<description>
 						<de>Wie entstanden die Songs? Und warum berührt er heute noch Menschen. Wer war das und wie war der Sound?</de>
-						<en>How were the songs produced? And why do the people still love the music? Who was it and how was the sound?</en>
+						<en>How did the songs originate? And why does he still touch people today. Who was he and what was the sound like?</en>
```

Comment:
why do they still (please correct German as well; review entire descriptions)

Reply:
I noticed that too but how do we know it was on-purpose and not an oversight by the original author? I tend to think "sloppiness" is on purpose for the laughs, same line in each episode.